prov/sm2/src/sm2.h

```diff
@@ -267,6 +268,8 @@ struct sm2_srx_ctx {
 	struct sm2_queue unexp_msg_queue;
 	struct sm2_queue unexp_tagged_queue;
 	struct sm2_recv_fs *recv_fs;
+
+	// TODO Determine if this spin lock is needed.
```

Comment:
I understand the addition of comments but this one isn't related to nemesis and adds unnecessary noise to this large patch.

Reply:
removed